sample-input/benchmarks/c5g7/c5g7-cmfd.py

```diff
@@ -318,28 +319,29 @@
 track_generator.setNumThreads(num_threads)
 track_generator.generateTracks()
 
+
 ###############################################################################
-###########################   Running a Simulation   ##########################
+#                            Running a Simulation
 ###############################################################################
 
-solver = CPUSolver(geometry, track_generator)
-solver.setSourceConvergenceThreshold(tolerance)
+solver = CPUSolver(track_generator)
+solver.setConvergenceThreshold(tolerance)
 solver.setNumThreads(num_threads)
-solver.convergeSource(max_iters)
+solver.computeEigenvalue(max_iters)
```

Comment:
Asking a question!  I'm not always good at finding the best option that lies between a FORTRAN77 style `DEIGPI` and a Java style `compute_that_eigenvalue_and_associated_eigenvalue_using_power_iteration`.  `computeEigenvalue` is fine with me, just wanted to make sure we adequately happy!